.changeset/popular-fishes-lay.md

```diff
@@ -0,0 +1,12 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+**fix: settings-related statistics not being updated according to license changes**
```

Comment:
pls focus on the changes (and how to present them) to the end user: https://developer.rocket.chat/contribute-to-rocket.chat/modes-of-contribution/participate-in-rocket.chat-development/development-workflow#id-4.-adding-changeset-to-your-pull-request
```suggestion
```